core/ustring.h

```diff
@@ -271,6 +272,7 @@ class String {
 	Vector<int> split_ints_mk(const Vector<String> &p_splitters, bool p_allow_empty = true) const;
 
 	String join(Vector<String> parts);
+	String join(List<String> parts);
```

Comment:
I'm not formally opposed to it if others in @godotengine/core think it's a good addition, but in general adding a new method and dependency to String just for a single debug message seems to go against our "solutions must be local" policy: https://docs.godotengine.org/en/latest/community/contributing/best_practices_for_engine_contributors.html#solutions-must-be-local

So I'd suggest either converting the list to a `Vector` or just using a `for` loop in place to construct the String to print.

Reply:
Sounds good, I'll add a local function for this.
It's a just a bit inconsistent that data structures with similar apis don't have the same capability.
Although a template approach might be appropriate here.